kratos/utilities/stl_io.h

```diff
@@ -24,14 +24,27 @@
 #include "includes/define.h"
 
 // Std::vecotr << operator
+
+namespace Kratos {
+
+// Std::vector << operator
 template<class T>
 std::ostream& operator<<(std::ostream& os, const std::vector<T> & data) {
 
-    std::cout << "[";
-    std::copy(data.begin(), data.end(), std::ostream_iterator<T>(std::cout, ", "));
-    std::cout << "[";
-    
+    os << "[";
+    const int vector_size = static_cast<int>(data.size());
+    if(vector_size) os << data[0];
```

Comment:
I personally don't like abusing ints as booleans, but your call
alternative would be: `if(vector_size>0) os << data[0];`